components/Layout/Header/Header.tsx

```diff
@@ -48,7 +48,10 @@ const HeaderComponent = ({
         loggedNavLinks
           .filter(({ name }) => name !== 'Forms in progress' || savedForms)
           .filter(
-            ({ name }) => name !== 'Manage workers' || user.hasDevPermissions
+            ({ name }) =>
+              name !== 'Manage workers' ||
+              user.hasDevPermissions ||
+              user.hasAdminPermissions
```

Comment:
Not sure I understand what's happening here 🤔 

I'm reading this as the list/array of `loggedNavLinks` is being filtered and would return elements that pass one of the conditions and the new filtered list would not contain 'Manage Workers' because of this condition `name !== 'Manage workers'` but that's not correct is it? 😅

Reply:
Yeah honestly this code is really really weird. I think it's saying "don't show `manage workers` unless you're a dev or admin.

Really, on second thought, I think we should modify how this works. Perhaps we should close this PR in favour of a more elegant solution! One for stand up in the morning

Reply:
👍🏾